src/styling/about.css

```diff
@@ -0,0 +1,26 @@
+hr {
+  border: 0;
+  clear: both;
+  display: block;
+  background-color: var(--main-yellow);
+  height: 3px;
+  margin-top: 5%;
```

Comment:
is there a px margin that would work? %s are a bit iffy bc it'll look different on different screens